client/add/add.ctrl.js

```diff
@@ -139,11 +139,19 @@ module.exports = class Add {
         );
         return lines;
       })
-      .then(lines => this.boxes.forEach((box) => {
+      .then(lines => {
```

Comment:
If we end up creating a map of ids to boxes, it would be better to iterate over `lines` rather than all of the user's boxes.